docs/logging.md

```diff
@@ -0,0 +1,59 @@
+# Logging
+
+Using Bigflow you can easily add handlers to your loggers that will send all loggers calls to the Cloud Logging.
+In the CL you can filter logs per project id/logger name and workflow id. 
+
+## Getting started
+To use logging you have to install `bigflow` with `logger` module, `bigflow[logger]`.
+To create logger with the CL handler you have to use `configure_logging` method.
```

Comment:
punctuation: To create logger with the CL handler, you have to use

Reply:
Done